sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -0,0 +1,239 @@
+import * as msRest from "@azure/ms-rest-js";
+import { ParsedKeyVaultEntityIdentifier } from "./core/keyVaultBase";
+import { CertificatePolicy } from "./core/models";
+
+export interface Certificate extends CertificateAttributes {
+  /**
+   * @member {string} [kid] The key id.
+   * **NOTE: This property will not be serialized. It can only be populated by
+   * the server.**
+   */
+  readonly kid?: string;
+  /**
+   * @member {string} [sid] The secret id.
+   * **NOTE: This property will not be serialized. It can only be populated by
+   * the server.**
+   */
+  readonly sid?: string;
+  /**
+   * @member {CertificatePolicy} [policy] The management policy.
+   * **NOTE: This property will not be serialized. It can only be populated by
+   * the server.**
+   */
+  readonly policy?: CertificatePolicy;
+  /**
+   * @member {Uint8Array} [cer] CER contents of x509 certificate.
+   */
+  cer?: Uint8Array;
+  /**
+   * @member {string} [contentType] The content type of the secret.
+   */
+  contentType?: string;
+}
+
+export interface CertificateAttributes extends ParsedKeyVaultEntityIdentifier {
+  /**
+   * @member {string} [id] The certificate id.
+   */
+  id?: string;
+  /**
+   * @member {boolean} [enabled] Determines whether the object is enabled.
+   */
+  enabled?: boolean;
+  /**
+   * @member {Date} [notBefore] Not before date in UTC.
+   */
+  notBefore?: Date;
+  /**
+   * @member {Date} [created] When the certificate was created.
+   */
+  created?: Date;
+  /**
+   * @member {Date} [updated] When the object was updated.
+   */
+  updated?: Date;
+  /**
+   * @member {Date} [expires] Expiry date in UTC.
+   */
+  expires?: Date;
```

Comment:
in the context of certificates I believe notBefore, expires, created and update are all readonly.